install-dev/data/db_structure.sql

```diff
@@ -1394,8 +1394,8 @@ CREATE TABLE `PREFIX_order_cart_rule` (
   `id_cart_rule` int(10) unsigned NOT NULL,
   `id_order_invoice` int(10) unsigned DEFAULT '0',
   `name` varchar(254) NOT NULL,
-  `value` decimal(17, 2) NOT NULL DEFAULT '0.00',
-  `value_tax_excl` decimal(17, 2) NOT NULL DEFAULT '0.00',
+  `value` decimal(20, 6) NOT NULL DEFAULT '0.000000',
+  `value_tax_excl` decimal(20, 6) NOT NULL DEFAULT '0.000000',
```

Comment:
It's a BC if people had 
```
123456789012345.00
```
You will have an error:
```
#1264 - Out of range value for column 'value' at row 1
```

Reply:
But this column is managed programmatically anyway when a discount is applied on Order, not manually
Besides if you look a the order table in the same file you can see (amongst other columns) ``total_paid_tax_incl` decimal(20, 6) NOT NULL DEFAULT '0.00',`
So the order won't be able to manage this kind of values either, it makes sens that the discount has the same range no?

Reply:
This table can be populate by a module or an external resource.
Even if other fields are already in `DECIMAL(20, 6)` this one can fail if someone use it